ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -305,14 +317,26 @@ TFLiteInterpreter::loadModel (int num_threads, accl_hw accelerator)
     interpreter->SetNumThreads (n);
   }
 
-  /** set nnapi delegate when accelerator set to auto (cpu.neon in Android), GPU or NPU */
-  if (accelerator == ACCL_CPU_NEON || accelerator == ACCL_GPU ||
-          accelerator == ACCL_NPU) {
+  /** set delegate after the accelerator prop */
+  if (accelerator == ACCL_CPU_NEON || accelerator == ACCL_NPU) {
 #if defined(__ANDROID__) && (TFLITE_VERSION_MAJOR >= 2 || TFLITE_VERSION_MINOR >= 14)
+    /** set nnapi delegate when accelerator set to auto (cpu.neon in Android) or NPU */
     stateful_nnapi_delegate.reset (new tflite::StatefulNnApiDelegate ());
     setDelegate (stateful_nnapi_delegate.get ());
 #else
     ml_logw ("NNAPI delegate is available only in Android with tflite v1.14.0 or higher");
+#endif
+  } else if (accelerator == ACCL_GPU) {
+#if defined(__ANDROID__) && (TFLITE_VERSION_MAJOR >= 2)
+    /** set gpu delegate when accelerator set to GPU */
+    TfLiteGpuDelegateOptionsV2 options = TfLiteGpuDelegateOptionsV2Default();
+    options.experimental_flags = TFLITE_GPU_EXPERIMENTAL_FLAGS_NONE;
+    options.experimental_flags |= TFLITE_GPU_EXPERIMENTAL_FLAGS_ENABLE_QUANT;
+
+    gpu_delegate.reset (TfLiteGpuDelegateV2Create (&options));
+    setDelegate (gpu_delegate.get ());
+#else
+    ml_logw ("GPU delegate is available only in Android with tflite v2.3.0");
```

Comment:
If gpu delegate is available after tflite 2.3.0, is it okay to check TFLITE_VERSION_MAJOR only?

Reply:
In https://github.com/nnstreamer/nnstreamer-android-resource/tree/master/external, we provide v2.3.0 only whose major version equals 2. So I thought the major version check would be sufficient.

Reply:
Well.. then it requires code changes whenever libraries with different version are added. I recommend you to consider forward-compatibility.

Reply:
You're right, I updated the log message with `v2.3.0 or higher`. Thanks!